modules/apps/asset/asset-categories-navigation-web/src/main/java/com/liferay/asset/categories/navigation/web/internal/display/context/AssetCategoriesNavigationDisplayContext.java

```diff
@@ -285,7 +294,36 @@ private String[] _getAssetVocabularyIds() {
 			}
 		}
 
-		return ArrayUtil.toStringArray(assetVocabularyIds);
+		return assetVocabularyIds;
+	}
+
+	private List<Long> _getLocalAssetVocabularyIds(
+		PortletPreferences portletPreferences) {
+
+		List<Long> assetVocabularyIds = new ArrayList<>();
+
+		String[] assetVocabularyExternalReferenceCodes =
+			GetterUtil.getStringValues(
+				portletPreferences.getValues(
+					"assetVocabularyExternalReferenceCodes", null));
+
+		for (String assetVocabularyGroupExternalReferenceCode :
```

Comment:
Shouldn't this variable be named `assetVocabularyExternalReferenceCode`? (without the "Group").

Reply:
You are right, thanks @adolfopa! I changed it